src/modules/admin/navigations/services/queries.gql.ts

```diff
@@ -0,0 +1,132 @@
+export const FIND_NAVIGATIONS = `
+	query getNavElements {
+		cms_navigation_element(distinct_on: placement, order_by: { placement: asc }) {
+			id
+			description
+			placement
+			tooltip
+		}
+		cms_navigation_element_aggregate(distinct_on: placement) {
+			aggregate {
+				count
+			}
+		}
+	}
+`;
+
+export const FIND_NAVIGATION_BY_PLACEMENT = `
+	query getMenuItemsByPlacement($placement: String!) {
+		cms_navigation_element(
+			order_by: { position: asc }
+			where: { placement: { _eq: $placement } }
+		) {
+			id
+			created_at
+			description
+			user_group_ids
+			icon_name
+			label
+			link_target
+			placement
+			position
+			updated_at
+			content_type
+			content_path
+			tooltip
+		}
+		cms_navigation_element_aggregate(where: { placement: { _eq: $placement } }) {
+			aggregate {
+				count
+			}
+		}
+	}
+`;
+
+export const FIND_NAVIGATION_BY_ID = `
+	query getMenuItemById($id: uuid!) {
+		cms_navigation_element(where: { id: { _eq: $id } }) {
+			id
+			created_at
+			description
+			user_group_ids
+			icon_name
+			label
+			link_target
+			placement
+			position
+			updated_at
+			content_type
+			content_path
+			tooltip
+		}
+	}
+`;
+
+export const GET_ALL_NAVIGATION_ITEMS = `
+	query getNavigationItems {
+		cms_navigation_element {
```

Comment:
you could also add ordering on the position here, so the frontend won't need to sort the items anymore.